src/frontend/components/VideoPlayer/VideoPlayer.tsx

```diff
@@ -49,8 +51,10 @@ export class VideoPlayer extends React.Component<
 
   constructor(props: VideoPlayerProps) {
     super(props);
-
-    this.state = { player: undefined };
+    this.state = {
+      isDashSupported: isMSESupported(),
```

Comment:
Why setting it in the state ? I don't see where you change its value.

Reply:
I can see 2 other ways:
- put it on a `const` floating in the module. This is fine per se but it's annoying to test as I'd have to dynamically re-import the module in my tests.
- Put it on `this`. The fact that a component is described by a `class` is an implementation detail and I don't want to depend on it.

I don't think there's any penalty to putting it on the state: since it never changes, it will never trigger re-renders and it's a clean place to put it.

Reply:
I'm not really convinced but using the state do the job so let's go merge it like this.